astropy/cosmology/_io/ecsv.py

```diff
@@ -24,8 +24,8 @@
     # datatype:
     # - {name: name, datatype: string}
     ...
-    # meta: !!omap
```

Comment:
What does this mean for people with older ESCV files with this `!!omap` though? Will their files no longer load with this patch? I think this was discussed before elsewhere but I don't have time to dig it up right now.

Reply:
Yeah removing that was a bad idea 😅 

This is continued in https://github.com/astropy/astropy/pull/16250 (closing this PR).